Symfony/CS/Fixer/Symfony/AliasFunctionsFixer.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\Symfony;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * @author Vladimir Reznichenko <kalessil@gmail.com>
+ * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ */
+class AliasFunctionsFixer extends AbstractFixer
+{
+    /**
+     * @var string[] stores alias (key) - master (value) functions mapping
+     */
+    public static $aliases = array(
```

Comment:
Ok, then we back to note from @GrahamCampbell - he also asked about this.
Currently I'm confused, and confusing people.

It was a public method, so UT could access mapping (UT generates test cases out of mapping) and not duplicate the map - removed as requested.

How do we proceed then?